loki/bulk/scheduler.py

```diff
@@ -55,6 +55,13 @@ def __init__(self, default, routines, disable=None, dimensions=None, dic2p=None,
             self.routines = CaseInsensitiveDict(routines)
         else:
             self.routines = CaseInsensitiveDict((r.name, r) for r in as_tuple(routines))
+        for routine in self.routines:
```

Comment:
Is this a subtle way of providing transformation config via the config file by storing it as values to a routine with name `trafo`? I would argue that we should implement #29 and thus have a proper way of providing such configs rather than using a hacky kind of implementation like this.